src/Task.gren

```diff
@@ -9,7 +9,7 @@ effect module Task where { command = MyCmd } exposing
 HTTP requests or writing to a database.
 
 
-@docs Task, perform, attempt, execute
+@docs Task, perform, attempt, execute, executeCmd
```

Comment:
So...

We have `perform` and `attempt`, which executes a `Task` _and_ sends the final value as a message to the app.
Then we have `execute` which simply executes the task.

I guess `executeCmd` is ok? 🤔

Reply:
I think having `execute` in the name is right as it's a 'fire and forget' task where no messages are sent back to the app. What about `executeWithCmd` to be more explicit?

Reply:
We've all thought a bit on this and concluded `executeCmd` makes the most sense :)